whamcloud/iml-docker
# Extract Postgres into its own docker container

Fixes #1 

Instead of having a single container with the entire manager jammed in it, use docker-compose / docker-stack to split out postgres to it's own container. There is an official one on docker hub we can try to start: https://hub.docker.com/_/postgres/.

To test:

```
docker build -t iml-docker-test .
docker-compose up
docker exec -it docker_iml_1 bash
$ ping postgres # should work fine
$ yum install firewalld
$ ping postgres # should still work
$ exit
docker restart docker_iml_1
docker exec -it docker_iml_1 bash
$ ping postgres # doesn't work
$ ping google.com # doesn't work, can't use nameservice resolution
```

Signed-off-by: Will Johnson <wjohnson@whamcloud.com>
